docs/quickstart/quickstart-docker.rst

```diff
@@ -0,0 +1,200 @@
+.. _ksql_quickstart_docker:
+
+Running KSQL Using Docker
+=========================
+
+This topic describes how to setup a Kafka cluster and start KSQL in a Docker container. After you complete these steps,
+you can return to the :ref:`create-a-stream-and-table` and start querying the data in the Kafka cluster.
+
+.. contents:: Contents
+    :local:
+    :depth: 1
+
+**Prerequisites**
+
+- Docker
+     - `macOS <https://docs.docker.com/docker-for-mac/install/>`__
+     - `All platforms <https://docs.docker.com/engine/installation/>`__
+- `Git <https://git-scm.com/downloads>`__
+- Java: Minimum version 1.8
+
+Start a Kafka cluster
+---------------------
+
+1. Clone the Confluent KSQL repository.
+
+   .. code:: bash
+
+       $ git clone git@github.com:confluentinc/ksql.git
+
+2. Change directory to the quickstart and launch the KSQL quick start in
+   Docker.
+
+   .. code:: bash
+
+       $ cd ksql/docs/quickstart/
+       $ docker-compose up -d
+
+   After you have successfully started the Kafka cluster and started
```

Comment:
This paragraph and the KSQL prompt doesn't belong here.  You only see the KSQL ascii art prompt after you ran the `docker-compose exec ksql-cli ksql-cli local --bootstrap-server kafka:29092` command in the next section on "Start KSQL".

Please also double-check that the non-docker Quickstart doesn't have the same mistake.